pkg/web_css/lib/src/_list.scss

```diff
@@ -56,7 +56,8 @@
   font-size: 12px;
   float: right;
   display: block;
-  margin-top: 4px;
+  margin: 4px 0px 4px 4px;
+  max-width: 32ex;
```

Comment:
This works out to around 32 characters right?

Reply:
yes